js/src/get-started-page/features-card/index.scss

```diff
@@ -1,27 +1,78 @@
 .gla-get-started-features-card {
-	.components-flex {
-		justify-content: space-around;
+	// Adjust <CardHeader> imported from @wordpress/components.
+	// Repeat selector to make it higher priority.
+	.components-card__header.components-card__header {
+		flex-direction: column;
+		gap: $grid-unit-10;
+		padding: $grid-unit-60 $grid-unit * 14.125 $grid-unit-50;
+		text-align: center;
 
 		@media (max-width: $break-small) {
-			flex-direction: column;
+			padding: $grid-unit-30 $grid-unit-30 $grid-unit-20;
+			gap: $grid-unit-20;
 		}
+	}
+
+	.components-flex {
+		gap: 50px;
+		padding: 0 $grid-unit * 11 $grid-unit-60;
 
 		&__block {
 			display: flex;
 			flex-direction: column;
 			align-items: center;
-			max-width: 195px;
-			margin: calc(var(--large-gap) / 4 * 5) 0;
```

Comment:
💅 
After replacing the margin spacing with `gap` style, we could also add `gap={ 0 }` prop to [the \<Flex\> component](https://github.com/woocommerce/google-listings-and-ads/blob/de6cd83cd0575f7a6fc257b710612b617dcfd290/js/src/get-started-page/features-card/index.js#L68). So that there will not have the default `margin-right` between \<FlexBlock\> components.

![2022-04-25 13 37 41](https://user-images.githubusercontent.com/17420811/165026995-8524ae3a-6d95-40d8-9d7a-0caacba22238.png)

Reply:
Thanks! Updated in c1753ec.